pip/tests-integration/resources/output/Adaptive_RI/ArithmeticOps.ll

```diff
@@ -104,16 +104,10 @@ attributes #1 = { "irreversible" }
 
 ; module flags
 
-!llvm.module.flags = !{!0, !1, !2, !3, !4, !5, !6, !7, !8, !9, !10}
+!llvm.module.flags = !{!0, !1, !2, !3, !4}
 
 !0 = !{i32 1, !"qir_major_version", i32 1}
 !1 = !{i32 7, !"qir_minor_version", i32 0}
 !2 = !{i32 1, !"dynamic_qubit_management", i1 false}
 !3 = !{i32 1, !"dynamic_result_management", i1 false}
-!4 = !{i32 1, !"classical_ints", i1 true}
-!5 = !{i32 1, !"qubit_resetting", i1 true}
-!6 = !{i32 1, !"classical_floats", i1 false}
-!7 = !{i32 1, !"backwards_branching", i1 false}
-!8 = !{i32 1, !"classical_fixed_points", i1 false}
-!9 = !{i32 1, !"user_functions", i1 false}
-!10 = !{i32 1, !"multiple_target_branching", i1 false}
+!4 = !{i32 1, !"int_computations", !"i64"}
```

Comment:
Will this change in module flags metadata name from `classical_ints` to `int_computations` be a breaking change anywhere in our existing flow (including any service validation)? Hopefully Quantinuum can already handle either, as they helped write the spec, but maybe worth a check before changing.

Reply:
To my knowledge this won't affect anything and PyQIR has no idea whether either exist as the adaptive profile work hasn't been started. @swernli @cesarzc care to weigh in?

Reply:
Yup, you're right, this won't affect any consumers as no one uses these flags yet. Now is the time to make the fix so they are consistent with the spec.